lib/sshkit/configuration.rb

```diff
@@ -29,6 +29,10 @@ def default_runner
       @default_runner ||= :parallel
     end
 
+    def default_runner_config
+      @default_runner_config ||= {}
```

Comment:
Perhaps it would be clearer to initialize this to `{ in: SSHKit.config.default_runner }` here?